johnsnowlabs/utils/env_utils.py

```diff
@@ -104,12 +104,15 @@ def is_running_in_emr():
 
 
 def is_running_in_databricks():
```

Comment:
```suggestion
def is_running_in_databricks_runtime():
```

We also might want to rename this as well for clarity.